src/command/process_update_task.rs

```diff
@@ -137,6 +172,16 @@ impl UpdateProcessor {
                 Ok(())
             }
             Command::Start => self.start_message().await,
+            Command::CurrentDefaultCity => {
+                let text = match self.chat.default_city_id {
+                    Some(id) => match self.repo.search_city_by_id(&id).await {
```

Comment:
maybe you should preload (with left join) default city when fetching a task so you don't have to make one more request when fetching the default city

Reply:
We will do #32